docs/servers/add-azure-server.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Add a Server to Azure
```

Comment:
The current name human readable for this credential provider is "Microsoft Azure".

I believe the title should include the provider's full name.

Could you apply the diff below?

```suggestion
title: Add a Server to Microsoft Azure
```